workers/unity/Packages/com.improbable.gdk.core/Utility/EntityTemplate.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using Improbable.Worker.Core;
+
+namespace Improbable.Gdk.Core
+{
+    public class EntityTemplate
+    {
+        internal readonly Entity template; //Internal for tests
```

Comment:
This can be replaced with a GetComponentIds function. Then we can remove the comment.

Reply:
Discussed offline - this can be disposable